tests/test_core.py

```diff
@@ -270,11 +270,11 @@ def test_dataset_errors():
     with pytest.raises(AttributeError):
         d.choice_multitrack()
 
-    d = mirdata.initialize("acousticbrainz_genre")
+    d = mirdata.initialize("orchset")
```

Comment:
no version="test', here?

Reply:
I don't think so, because here, if I'm not mistaken, we are testing the thrown errors, and in this particular one, we are trying to confirm that if you initialize a dataset version (a full one), and you do not download the index, this will be missing, and therefore you need to throw a  `FileNotFoundError` if trying to get it by running `dataset._index`. You are basically testing [these lines.](https://github.com/mir-dataset-loaders/mirdata/blob/97a2e080c85f068b06593e1719115abc2d2fe6b4/mirdata/core.py#L169C1-L177C14)